### PR TITLE
Creating and building an application

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR:

* Enables to use Windows in combination with Cygwin to create and build an app
* Improves the DB creation doc (notably enabling PostGIS)
* Make a built app use c2cgeoform as a wheel (and not a local package)

Feel free to merge or dismiss this PR, but be aware that I have not tested the modification off the Make files on Linux.

### CODE REVIEW

**Comment by kalbermattenm** [comment]
Hmmm... my friend Travis...

image

I do not get this error. If anyone has a clue why Travis sees it as recursive, I will be happy to learn something...?

File: c2cgeoform/scaffolds/c2cgeoform/Makefile_tmpl [hunk 1]
```diff
@@ -1,5 +1,14 @@
 MO_FILES = $(addprefix {{package}}/locale/, fr/LC_MESSAGES/{{package}}.mo de/LC_MESSAGES/{{package}}.mo)
 
+ifneq (,$(findstring CYGWIN, $(shell uname)))
+PYTHON3 =
+VENV_BIN = .build/venv/Scripts
+PIP_UPGRADE = python.exe -m pip install --upgrade pip setuptools
+else
+PYTHON3 = $(PYTHON3)
```
Comment (arnaud-morvan) [inline c2cgeoform/scaffolds/c2cgeoform/Makefile_tmpl]: Here is the variable that references itself, same for VENV_BIN

File: docs/create-project.md [hunk 2]
```diff
@@ -58,7 +68,8 @@ know how to do that.
 Create the database :
 
 ```shell
-sudo -u postgres createdb myproject -T template_postgis
+sudo -u postgres createdb myproject
+sudo -u postgres psql -c 'CREATE EXTENSION postgis;'
```
Comment (arnaud-morvan) [inline docs/create-project.md]: missing `-d myproject`

**Comment by arnaud-morvan** [comment]
Manually merged in master.

**Pull request closed without merging**